frontend/projects/upgrade/src/app/core/experiments/store/experiments.model.ts

```diff
@@ -354,3 +354,13 @@ export interface InteractionEffectGraphData {
   series: InteractionEffectLineChartSeriesData[];
   dot: boolean;
 }
+
+export interface ValidateExperimentError {
```

Comment:
Are these types used in other places apart from the component? If not, we should move it inside the component.

Reply:
code updated